staging/src/k8s.io/client-go/examples/workqueue/BUILD

```diff
@@ -1,7 +1,7 @@
 package(default_visibility = ["//visibility:public"])
 
 load(
-    "@io_bazel_rules_go//go:def.bzl",
+    "//build:go.bzl",
```

Comment:
Can these changes get a separate PR, just to make this easier to scan - thousands of files, many with justthis change

Reply:
Check out https://github.com/kubernetes/kubernetes/pull/62823